### PR TITLE
Added consistent SpinFadeCube loaders across app

### LINKED ISSUE #25: Update all loaders to use Splash screen's loader
The splash screen uses a `Spining Cube loader`, while the rest of the app uses a simple `Circular Loader`

Change every loader in the app to use the Spinning Cube loader from splash screen.

### PR DESCRIPTION
**Pull request by ibrahimbroachwala**
Closes #25  
1) Used loader from Splash Screen across app.
2) Changed color of Loader to AppTheme's primary color for lighter backgrounds.
3) Reduced loader size to 40 for page loaders.

### CODE REVIEW

File: ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8,12 +8,8 @@
 
 /* Begin PBXBuildFile section */
 		1498D2341E8E89220040F4C2 /* GeneratedPluginRegistrant.m in Sources */ = {isa = PBXBuildFile; fileRef = 1498D2331E8E89220040F4C2 /* GeneratedPluginRegistrant.m */; };
-		2D5378261FAA1A9400D5DBA9 /* flutter_assets in Resources */ = {isa = PBXBuildFile; fileRef = 2D5378251FAA1A9400D5DBA9 /* flutter_assets */; };
```
Comment (mannprerak2) [inline ios/Runner.xcodeproj/project.pbxproj:11]: Did you test this application on an iOS device?

File: lib/ui/tabs/discover_tab.dart [hunk 2]
```diff
@@ -46,6 +47,12 @@ class DiscoverTab extends StatelessWidget {
           }
         });
       },
+      loadingBuilder: (context) {
+        return SpinKitFadingCube(
+          color: Color(0xffB54646),
```
Comment (mannprerak2) [inline lib/ui/tabs/discover_tab.dart]: You should never hard code color values like this, but instead, create a separate file to handle all these color constants with proper names denoting where they are used, or set colors of your theme in the MaterialApp widget.

For now, a simple fix is to replace all these `Color(0xffB54646)` with `Theme.of(context).primaryColor`.
Please make these changes everywhere you have hardcoded the color value.

File: lib/ui/tabs/event_page.dart [hunk 3]
```diff
@@ -85,7 +86,10 @@ class EventDetailPageState extends State<EventDetailPage> {
                     ),
                   ),
                   body: Center(
-                    child: CircularProgressIndicator(),
+                    child: SpinKitFadingCube(
+                      color: Color(0xffB54646),
+                      size: 50.0,
```
Comment (mannprerak2) [inline lib/ui/tabs/event_page.dart:91]: The same also holds true for size, never hardcode size of something like this. Always create a separate file for your style constants.

But, for now, let's leave it hardcoded.

**Review by mannprerak2 — CHANGES_REQUESTED** [review]

File: lib/ui/tabs/discover_tab.dart (hunk 2 above)
Comment (ibrahimbroachwala) [reply lib/ui/tabs/discover_tab.dart]: Sure. Guess this repo also needs another PR for cleaning the entire code and creating separate style files.

File: ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ibrahimbroachwala) [reply ios/Runner.xcodeproj/project.pbxproj:11]: Yes I ran first in an iOS emulator. That's why you see all the changes in ios/ folder. 

I guess I should have removed them from git, My bad. But it does no harm.

File: lib/ui/tabs/discover_tab.dart (hunk 2 above)
Comment (mannprerak2) [reply lib/ui/tabs/discover_tab.dart]: Yes, you can make another PR for this if you like

File: ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mannprerak2) [reply ios/Runner.xcodeproj/project.pbxproj:11]: No need to remove these files

File: lib/ui/tabs/discover_tab.dart (hunk 2 above)
Comment (ibrahimbroachwala) [reply lib/ui/tabs/discover_tab.dart]: Please check updated code.

**Review by mannprerak2 — APPROVED** [review]

**Comment by mannprerak2** [comment]
Thank you for your contribution!

**Pull request merged**